src/gkApi.ts

```diff
@@ -138,7 +159,12 @@ export const fetchProviderToken = async (provider: Provider) => {
 	}
 
 	const payload = await res.json();
-	return payload.data as ProviderToken;
+	const providerToken = payload.data as ProviderToken;
+	if (providerToken.expiresIn < 0 && providerToken.type === 'oauth' && provider !== 'github') {
```

Comment:
wondering why we are excluding github here explicitly?

Reply:
GitHub tokens don't need to be refreshed, they don't expire

Reply:
I can add a comment

Reply:
ah it's just the way it's written is weird, cause it checks github at the end after it already checks `expiresIn`